img/c/constellix.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90"><circle cx="41" cy="49" r="41" fill="#FFF"/><circle cx="41" cy="49" r="26" fill="#424B93"/><path fill="#424B93" d="M86.06 0a4.9 4.9 0 00-1.35.29l-.3.1c-.48.21-.97.48-1.42.8l-.2.16-.46.32-.98.69-.97.7-1.44 1.05-.94.72-1.4 1.09-.93.74-.91.75c-.77.62-1.52 1.27-2.27 1.92l-.9.8c-.6.52-1.2 1.06-1.79 1.61l-.89.84-.88.85c-.6.57-1.18 1.15-1.77 1.74l-.89.9-.44.45-.89.92-.44.47-.89.95-.9.97-.89 1-.45.5-.9 1.02-.9 1.04-.46.53-.92 1.08-.92 1.1-.93 1.12-.93 1.14-.95 1.17-.96 1.2-.96 1.22-.98 1.25-.99 1.27-1 1.3-1.01 1.34-1.03 1.36-1.04 1.39-1.05 1.42-1.07 1.45L41.8 45l-10.02-8.48a14.01 14.01 0 00-3.85-2.51 7.1 7.1 0 00-4.28-.55 4.51 4.51 0 00-3.61 3.95 6.61 6.61 0 001.13 4.23l.2.3.07.11 14.54 19.9.48.65.42.55c.68.87 1.28 1.54 1.86 2.08 1.27 1.18 2.61 1.85 4.23 1.75 1.63-.1 2.88-.94 3.96-2.27.28-.34.56-.73.84-1.18l.3-.47.15-.25.32-.55.16-.3.35-.62C59.54 43.16 72.7 25.15 88.53 6.79c.5-.5.9-1.07 1.18-1.73.24-.6.36-1.26.25-1.98-.14-1-.7-1.88-1.53-2.44A3.67 3.67 0 0086.28 0h-.22z"/><path fill="#FFF" d="M38.66 59.94l-14.5-19.85c-.6-.9-.85-1.7-.78-2.29.03-.28.14-.51.3-.7.18-.17.41-.3.7-.36a3.9 3.9 0 012.24.37c.94.4 1.95 1.05 2.94 1.95L42.4 49.9l2.16-2.96 1.06-1.47 1.06-1.43 1.55-2.09 1.01-1.35 1.5-1.98.98-1.29.97-1.26.96-1.23.94-1.2.94-1.19.93-1.15.91-1.13.91-1.1.9-1.1.9-1.05.44-.53.88-1.02.88-1.01.44-.5.87-.97.44-.48.86-.95.43-.47.86-.92.86-.9.86-.88.86-.87.85-.84.86-.84.86-.82.86-.8.86-.8.86-.77.87-.76.87-.75.88-.74.88-.73.89-.72.9-.7.9-.7.9-.69.93-.68.92-.67.47-.34.95-.66.47-.33.1-.07c.32-.26.68-.46 1-.57.16-.06.3-.09.42-.1.13 0 .22.02.29.06a.2.2 0 01.09.15c0 .07 0 .15-.04.24-.08.2-.26.44-.55.74C70.01 23.1 56.74 41.29 46.1 59.72l-.29.53-.27.49-.26.43-.23.4-.12.17-.21.33-.1.15-.2.27c-.04.04-.08.08-.1.12-.55.67-1.04 1-1.56 1.03-.51.03-1.07-.24-1.73-.85-.24-.23-.51-.51-.8-.85l-.3-.36-.32-.4-.17-.21-.36-.47-.42-.56z"/></svg>
```

Comment:
Although the favicon has an outer white circle, they don't seem to use that anywhere else:
![image](https://user-images.githubusercontent.com/20560218/127356097-c2588b7d-3195-4068-a2b4-3e691d04f6b8.png)
![image](https://user-images.githubusercontent.com/20560218/127356194-f20b3cc7-50cf-4af7-bcbb-d7c2e8e0af22.png)

I think it may be better to remove the outer circle.

Reply:
done